cli/lib/kontena/cli/vpn/config_command.rb

```diff
@@ -7,7 +7,8 @@ def execute
       require 'rbconfig'
       require_api_url
       payload = {cmd: ['/usr/local/bin/ovpn_getclient', 'KONTENA_VPN_CLIENT']}
-      stdout, stderr = client(require_token).post("services/#{current_grid}/vpn/server/containers/1/exec", payload)
+      service = client(require_token).get("services/#{current_grid}/vpn/server/containers")['containers'][0]
+      stdout, stderr = client(require_token).post("containers/#{service['id']}/exec", payload)
```

Comment:
Is this really right? Shouldn't the path be something like `containers/<node>/vpn-server-1/exec`